openshift-client/src/main/java/io/fabric8/openshift/client/dsl/internal/ProjectOperationsImpl.java

```diff
@@ -51,4 +64,114 @@ public ProjectOperationsImpl newInstance(OperationContext context) {
   public boolean isResourceNamespaced() {
     return false;
   }
+
+  @Override
+  public List<HasMetadata> createProjectAndRoleBindings(String name, String description, String displayName, String adminUser, String requestingUser) {
+    List<HasMetadata> result = new ArrayList<>();
+    Project project = getProject(name, description, displayName, requestingUser);
+    List<HasMetadata> projectRoleBindings = getNewProjectRoleBindings(name, adminUser);
+
+    // Create Project
+    ProjectOperationsImpl projectOperations = new ProjectOperationsImpl(client, OpenShiftConfig.wrap(config));
+    result.add(projectOperations.create(project));
+
+    // Create Role Bindings
+    NamespaceVisitFromServerGetWatchDeleteRecreateWaitApplicableListImpl listOp = new NamespaceVisitFromServerGetWatchDeleteRecreateWaitApplicableListImpl(client, config, getNamespace(), null, false, false, new ArrayList<>(), projectRoleBindings, null, DeletionPropagation.BACKGROUND, true) {};
+    result.addAll(listOp.createOrReplace());
+
+    return result;
+  }
+
+  private Project getProject(String name, String description, String displayName, String requestingUser) {
```

Comment:
nit: Maybe calling this `initProject` would be less confusion. By the name of the method I'm unsure if this will retrieve a project from the server or something in those lines.